src/http/server.rs

```diff
@@ -80,6 +81,15 @@ pub async fn run_server(config_path: &PathBuf) -> std::io::Result<()> {
     let ayb_conf_for_server = ayb_conf.clone();
     fs::create_dir_all(&ayb_conf.data_path).expect("Unable to create data directory");
     let ayb_db = connect_to_ayb_db(ayb_conf.database_url).await.unwrap();
+    let web_info = if let Some(web_conf) = ayb_conf.web {
+        Some(
+            WebInfo::from_url(&web_conf.info_url)
+                .await
+                .expect("failed to retrieve information from the web frontend"),
```

Comment:
```suggestion
                .expect("Failed to retrieve information from the web frontend"),
```

Reply:
Expect messages should be undercased, that's how std and most libraries do it
Example: https://doc.rust-lang.org/std/option/enum.Option.html#method.expect

Reply:
I learn something new every day! Thank you! :)